packages/frontend/web-ui/src/auth/hooks/useRegisterForm.ts

```diff
@@ -0,0 +1,151 @@
+import { useEffect, useState } from 'react';
+import { validateName } from '../../common/helpers/validateName';
+import { validateEmail } from '../../common/helpers/validateEmail';
+import { validatePassword } from '../../common/helpers/validatePassword';
+import { validateConfirmPassword } from '../../common/helpers/validateConfirmPassword';
+import { httpClient } from '../../common/http/services/HttpService';
+import { buildSerializableResponse } from '../../common/http/helpers/buildSerializableResponse';
+import { Either } from '../../common/models/Either';
+import { UseRegisterFormParams } from '../models/UseRegisterFormResult';
+import { RegisterStatus } from '../models/RegisterStatus';
+import { FormValidationResult } from '../models/FormValidationResult';
+import { FormFieldsRegister } from '../models/FormFieldsRegister';
+import { RegisterResponse } from '../../common/http/models/RegisterResponse';
+import { RegisterSerializedResponse } from '../../common/http/models/RegisterSerializedResponse';
+
+export const INVALID_CREDENTIALS_REG_ERROR: string = 'Invalid credentials.';
```

Comment:
This feels wrong. It's being used as the error message when the create user endpoint returns a conflict status code. This response means there's a conflict with another entity, in this case a user. This means there's an existing user matching the one was attempted to be created.

```suggestion
export const HTTP_CONFLICT_ERROR_MESSAGE: string = 'Cannot register user. There's already a user with the same email address.';
```

Reply:
Done!